packages/error-boundry-fallback/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@opensrp/error-boundry-fallback",
+  "version": "0.0.1",
```

Comment:
This isn't published yet. Let's have it at **0.0.0**. Also, let's update the package.json to match [this template on the master branch](https://github.com/OpenSRP/web/blob/master/packages/template/package.json)